src/evidently/metric_preset/data_drift.py

```diff
@@ -90,3 +101,17 @@ def generate_metrics(self, data: InputData, columns: DatasetColumns):
                 per_column_stattest_threshold=self.per_column_stattest_threshold,
             ),
         ]
+        embeddings_data = data.column_mapping.embeddings
+        if embeddings_data is not None:
```

Comment:
if you invert the statement here, the code will be a little bit more readable. ie 
```
if data is None:
   return result

...code inside current if block
return result
```

https://medium.com/lemon-code/guard-clauses-3bc0cd96a2d3